src/vivarium/framework/engine.py

```diff
@@ -207,6 +208,11 @@ def __init__(
     def name(self) -> str:
         return self._name
 
+    @property
+    def current_time(self) -> Time:
```

Comment:
Is this actually used anywhere?

Reply:
It's used by VCT so that we can know how many steps are left in the restarted run.